### PR TITLE
User can read articles based on their geolocation

### PR DESCRIPTION
**Pull request by jennymalmefeldt**
# This PR contains:

## [PT](https://www.pivotaltracker.com/story/show/175328280)

- Req spec
- Adding column location to article model  
- Conditional for location params
- Adding location to seed file
- Bug fix: -article not created if no picture attached-

### CODE REVIEW

File: app/controllers/api/v1/admin/articles_controller.rb [hunk 1]
```diff
@@ -13,15 +17,8 @@ def create
...
-  def attach_image(article)
-    params_image = params[:article][:image]
-    if params_image.present?
-      DecodeService.attach_image(params_image, article.image)
-    end
-  end
-
   def article_params
-    params.require(:article).permit(:title, :teaser, :content, :category, :premium)
+    params.require(:article).permit(:title, :teaser, :content, :category, :premium, :location)
```
Comment (johanperjulius1) [inline app/controllers/api/v1/admin/articles_controller.rb:21]: should location be required? Then you can't create an article without location?

File: spec/requests/api/v1/articles/index_location.spec.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    end
+
+    it "is expected to respond with ok status" do
+      expect(response).to have_http_status :ok
+    end
+
+    it "is expected to return a specific article title" do
+      expect(response_json["articles"].first["title"]).to eq "This is France"
+    end
+
```
Comment (johanperjulius1) [inline spec/requests/api/v1/articles/index_location.spec.rb:53]: The location says successfully italy, but then the expected title is "This is France". Should it be like that? It is confusing?

**Review by johanperjulius1 — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/admin/articles_controller.rb (hunk 1 above)
Comment (ctl-lab) [reply app/controllers/api/v1/admin/articles_controller.rb:21]: it is required. Check the model out

File: spec/requests/api/v1/articles/index_location.spec.rb (hunk 2 above)
Comment (ctl-lab) [reply spec/requests/api/v1/articles/index_location.spec.rb:53]: It should be like that. Italy meaning outside of Sweden. Then you get non-Swedish articles. In this case something from France

**Review by johanperjulius1 — APPROVED** [review]

**Pull request merged**